src/features/submitCode/gitPush/ui/GitPushDialog.tsx

```
@@ -72,7 +75,7 @@ export default function GitPushDialog({ githubUrl }: GitPushDialogProps) {
           <div className="flex items-center justify-between">
             <label className="text-sm font-medium text-[#ccc]">자동 푸시</label>
             <Switch
-              checked={autoPushStatus}
+              checked={currentGitPushData?.gitPushStatus}
               onCheckedChange={() => pushAutoToggle()}
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)